lib/_stream_writable.js

```diff
@@ -198,7 +199,7 @@ ObjectDefineProperty(WritableState.prototype, 'buffer', {
   get: internalUtil.deprecate(function writableStateBufferGetter() {
     return this.getBuffer();
   }, '_writableState.buffer is deprecated. Use _writableState.getBuffer ' +
-     'instead.', 'DEP0003')
+    'instead.', 'DEP0003')
```

Comment:
I was initially planning to move this definition as well into the block. But then we need to add `set` since defineProperties would return `undefined`. I feel, if we do that, we can move out all our definition to a single block, which is easy to navigate and read. Thoughts? (Anyways, we can do that as a separate PR as well).

Reply:
Note. This is on the `WritableState` prototype.